package.json

```diff
@@ -41,8 +40,5 @@
     "husky": "^4.3.6",
     "lint-staged": "^10.5.3",
     "prettier": "^2.2.1"
-  },
-  "dependencies": {
-    "express": "^4.17.1"
```

Comment:
looks like you removed express?

Reply:
Yes, I removed express from the root repository package.json, since we created a backend app with express already installed

Reply:
the commit message says it removed nodemon... :)

Reply:
Just noticed... can you fix commit messages? Will look into it